cmd/config.go

```diff
@@ -129,16 +135,45 @@ $ %s cfg i`, appName, a.homePath, appName)),
 
 // GlobalConfig describes any global relayer settings
 type GlobalConfig struct {
-	Timeout     string `yaml:"timeout" json:"timeout"`
-	KMSKeyID    string `yaml:"kms-key-id" json:"kms-key-id"`
-	ClusterMode bool   `yaml:"cluster-mode" json:"cluster-mode"`
+	Timeout     string         `yaml:"timeout" json:"timeout"`
+	KMSKeyID    string         `yaml:"kms-key-id" json:"kms-key-id"`
+	ClusterMode *ClusterConfig `yaml:"cluster-mode" json:"cluster-mode"`
+}
+
+// SetClusterMode sets the cluster mode for the global config
+type ClusterConfig struct {
+	Enabled    bool   `yaml:"enabled" json:"enabled"`
+	Key        string `yaml:"key" json:"key"`
+	privateKey *ecdsa.PrivateKey
+}
+
+func (c ClusterConfig) IsEnabled() bool {
+	return c.Enabled
+}
+
+func (c ClusterConfig) SignMessage(msg []byte) ([]byte, error) {
```

Comment:
> I think these behaviours SignMessage and VerifySignature should be of the type Relayer rather than the config.

I didn't get, can you please elaborate further?